feature-libs/order/components/order-details/order-details-extended/consignment-tracking/consignment-tracking-link.component.ts

```diff
@@ -0,0 +1,77 @@
+/*
+ * SPDX-FileCopyrightText: 2023 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import {
+  ChangeDetectorRef,
+  Component,
+  ElementRef,
+  inject,
+  OnDestroy,
+  OnInit,
+  ViewChild,
+  ViewContainerRef,
+} from '@angular/core';
+import {
+  LaunchDialogService,
+  LAUNCH_CALLER,
+  OutletContextData,
+} from '@spartacus/storefront';
+import { of, Subscription } from 'rxjs';
+import { take } from 'rxjs/operators';
+import { Consignment } from '@spartacus/order/root';
+import { OrderConsignmentsService } from '../../order-consignments.service';
+
+@Component({
+  selector: 'cx-consignment-tracking-link',
+  templateUrl: './consignment-tracking-link.component.html',
+  host: { class: 'cx-list-header col-12' },
+})
+export class ConsignmentTrackingLinkComponent implements OnInit, OnDestroy {
+  consignment: Consignment;
+  protected subscription = new Subscription();
+  protected orderConsignmentsService = inject(OrderConsignmentsService);
+  protected launchDialogService = inject(LaunchDialogService);
+  protected vcr = inject(ViewContainerRef);
+  protected cd = inject(ChangeDetectorRef);
```

Comment:
[NITPICK]
Let's stick to convention of naming `cdr`, instead of `cd`

Reply:
removed this